package/src/components/MessageInput/SendButton.tsx

```diff
@@ -53,43 +56,4 @@ const SendButtonWithContext = (props: SendButtonPropsWithContext) => {
   );
 };
 
-const areEqual = (prevProps: SendButtonPropsWithContext, nextProps: SendButtonPropsWithContext) => {
```

Comment:
This change is because of React. The memo is causing yellow boxes on the profiler here on the text change. But without the memo, there are green boxes. I am trying to figure out what the cause is, but this is definitely noticeable.